code/game/objects/items/storage/pouch.dm

```diff
@@ -621,7 +621,7 @@
 /obj/item/storage/pouch/construction
 	name = "construction pouch"
 	desc = "It's designed to hold construction materials - glass/metal sheets, metal rods, barbed wire, cable coil, and empty sandbags. It also has a hook for an entrenching tool."
-	storage_slots = 3
+	storage_slots = 5
```

Comment:
storage creep.

Reply:
You 're not supposed to hold 150000 kh of mats on you

Reply:
don't see how this is an issue tho? you need this much space to hold the extra mats you will now get from this PR, and there aren't many options for engis except for jaeger engi (which forces you to go jaeger + slows you down)